six.py

```diff
@@ -0,0 +1,953 @@
+# Copyright (c) 2010-2019 Benjamin Peterson
```

Comment:
Wait, why is this included? `six` should be included in `requirements.txt` and installed via `pip`.

Reply:
Do we know if `edx` will have their own copy of `six` available? If not, we'll need to include it like we do for `voluptuous`, right?

Reply:
You know what? I'm going to ask the edX people right now...

Reply:
Actually, I just checked in studio - six is presently available to codejail as a standard python library. I assume it will be in python3 as well (if not, I'll raise hell about it).

Reply:
Ah, smart, I should have checked. Happy to not include it, then :)